src/main/app/src/app/zaken/besluit-create/besluit-create.component.ts

```diff
@@ -56,6 +66,10 @@ export class BesluitCreateComponent implements OnInit, OnDestroy {
         ingangsdatumField.formControl.valueChanges.pipe(takeUntil(this.ngDestroy)).subscribe(value => {
             (vervaldatumField as DateFormField).minDate = value;
         });
+        besluittypeField.formControl.valueChanges.pipe(takeUntil(this.ngDestroy)).subscribe(value => {
+            zoekparameters.toegestaneInformatieObjectTypeUUIDs = value.informatieObjectTypen.map(x => x.split("/").pop());
```

Comment:
Ik zie niet waar het veld informatieObjectTypen vandaan komt, maar het lijken op url's, die wil je niet van de achterkant naar de voorkant sturen en dan weer terug. Zo wordt het gevoelig voor een eventuele man-in-the-middle aanval. Wanneer je dus het besluittype in de zoekparameters toevoegd, heb je dit probleem ook niet.

Los daarvan, denk aan je naamgeving. Variabele x?

Reply:
Opgelost in opvolgende commit.

Wat betreft de URI's; ik haal de UUID uit de URI omdat er geen veld `UUID` beschikbaar is op deze objecten. Wat betreft de naamgeving van de variabelen; ik was onder de veronderstelling dat dit bij een simpele mapping op een lijst geen probleem was, maar ik houd er in de toekomst rekening mee.